src/clj/game/cards/resources.clj

```diff
@@ -1832,7 +1832,12 @@
   {:data {:counter {:power 4}}
    :events [(trash-on-empty :power)
             {:event :runner-spent-click
+             :once :per-turn
              :req (req (let [all-cards (get-all-cards state)]
+                         ;; TODO - assert this works when (e.g.) using the last click on lib-acc
+                         ;; and having it trashed (all-cards wont find it)
```

Comment:
Why won't all-cards find it?

Reply:
If the card gets trashed through use (ie last click on liberated accounts). I need to test this and check the behaviour, since I think things like that are where the issue actually is, and why some people aren't reproducing the bug and some are - and first-event is checking for cids using the current state.